azure-pipelines/azure-templates/docker-template.yml

```diff
@@ -0,0 +1,56 @@
+
+parameters:
+  - name: 'displayName'
+    type: string
+  - name: 'targetPath'
+    type: string
+  - name: 'artifact'
+    type: string
+  - name: 'tags'
+    type: string
+  - name: 'Dockerfile'
+    type: string
+  - name: 'repository'
+    type: string
+  - name: 'stage'
+    type: string
+
+stages:
+- stage: ${{ parameters.stage }}
+  displayName: ${{ parameters.displayName }}
+  dependsOn: Build_and_Test
+  jobs:
+  - job: BuildPushAzure
+    steps:
+
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        buildType: 'current'
+        artifactName: ${{ parameters.artifact }}
+        itemPattern: '**.jar'
+        targetPath: ${{ parameters.targetPath }}
+    - task: Docker@2
+      inputs:
+        containerRegistry:
+        repository: $(service)
+        command: 'buildAndPush'
+        Dockerfile: ${{ parameters.Dockerfile }}
+        tags: ${{ parameters.tags }}
+
+  - job: BuildPushDocker
+    steps:
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        buildType: 'current'
+        artifactName: ${{ parameters.artifact }}
+        itemPattern: '**.jar'
+        targetPath: './dockerize/'
+    - task: Docker@2
+      inputs:
+        containerRegistry:
+        repository: ${{ parameters.repository }}
+        command: 'buildAndPush'
+        Dockerfile: ${{ parameters.Dockerfile }}
+        tags: ${{ parameters.tags }}
```

Comment:
i can't really tell what the purpose of these two separate jobs is. add some comments

Reply:
Removed. Basically what happened with these were for reference, and forgot to git ignore them.